app/Resources/assets/css/application.scss

```diff
@@ -219,3 +219,25 @@ a.disabled {
         }
     }
 }
+
+// Panel macro styles. See also views/macros/layout.html.twig
+.panel {
+    .panel-heading {
+        display: flex;
+        align-items: end;
+        .panel-title {
+            font-size: 24px;
+            flex: 3;
+        }
+        .panel-description {
+            flex: 1;
```

Comment:
Only thing here is the text isn't vertically aligned. I accomplished this by setting an explicit `height` and `line-height` https://github.com/wikimedia/grantmetrics/pull/87/files#diff-ff1c57142581996a6cd7e089b450dc9fL112 so we might restore that.

There may be a better way to do it, though!

Reply:
I was going for the `end` alignment. But now you mention it, `center` is better! :) And no need for explicit height.

Oh, and it turns out that I'm confused: the [W3C]( https://www.w3.org/TR/css-flexbox-1/#propdef-align-items) says different things to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items). Anyway, center is what we want yeah?

Reply:
Yeah vertically centered, right-alighted.

Reply:
Done. (Sorry, I did the gerrit-world thing of modifying an existing commit, so GH probably didn't tell you about it.)